### PR TITLE
Parse email files attachments msg

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/44847

## Description
added attachment name in `msg` file

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_6_12.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### ParseEmailFiles
+ - Fixed an issue that the script does not show the attachment name in the war room.
```
Comment (reutshal) [inline Packs/CommonScripts/ReleaseNotes/1_6_12.md:4]:
```suggestion
 - Fixed an issue where the attachment name is not shown in the war room.
```

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 2]
```diff
@@ -3468,7 +3468,7 @@ def handle_msg(file_path, file_name, parse_only_headers=False, max_depth=3):
         'Text': msg_dict['Text'],
         'Headers': headers,
         'HeadersMap': headers_map,
-        'Attachments': '',
+        'Attachments':  msg_dict['Attachments'],
```
Comment (reutshal) [inline Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3471]:
```suggestion
        'Attachments':  msg_dict.get('Attachments'),
```
Could this be None?

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles_test.py [hunk 3]
```diff
@@ -814,6 +814,21 @@ def test_get_msg_mail_format():
     assert msg_mail_format == ''
 
 
+def test_handle_msg_with_attachments():
+    """
+    Given:
+     - A msg file with attachments
+
+    When:
+     - The handle_msg
```
Comment (reutshal) [inline Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles_test.py:823]:
```suggestion
     - Running the 'handle_msg' method
```

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 4]
```diff
@@ -2703,7 +2703,7 @@ def __init__(self, directory_entries, parent_directory_path=None):
     def _get_attachments_names(self):
         names = []
         for attachment in self.attachments:
-            names.append(attachment.DisplayName)
+            names.append(attachment.DisplayName or attachment.Filename)
```
Comment (reutshal) [inline Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:2706]: When will the attachment not have a DisplayName? 
Do we have a case like that in the test data?

**Review by reutshal — CHANGES_REQUESTED** [review]

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py (hunk 4 above)
Comment (MosheEichler) [reply Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:2706]: Yes (`html_attachment.msg` in `test_data` folder). And we're testing it

**Review by reutshal — APPROVED** [review]

**Pull request merged**